README.md

```diff
@@ -39,6 +43,25 @@ To deploy, run `now alias shields-raster-staging.now.sh raster.shields.io`
 
 The production server is at: https://raster.shields.io/
 
+## Support
+
+If you're having problems with the service, you can post questions in the
+[`#support` room in Discord][discord] and [the main Shields repo][new issue].
+
+## Contributing
+
+If you use and love Shields, we ask that you
+[make a one-time \$10 donation to Shields][opencollective]. You can also
+contribute monthly or sponsor at a higher level.
+
+Contributions welcome! Feel free to
+[open discussions on the main Shields repo][new issue] and to open pull requests here.
+
+The [`#contributing` room in Discord][discord] is also a good place to discuss and ask
+questions.
+
+[new issue]: https://github.com/badges/shields/issues/new/choose
+
 ## License
 
 This project is licensed under the MIT license.
```

Comment:
Would it be worth noting here that this repo is licensed separately/differently than the main Shields repo/app?

Reply:
I don't think it's necessary since what you can do is effectively very similar.

To be honest I'd like to relicense Shields under something like MIT because it gives contributors explicit legal protection.

Reply:
SGTM, on both fronts